test/functional/test_framework/test_node.py

```diff
@@ -115,6 +115,22 @@ def get_deterministic_priv_key(self):
         ]
         return PRIV_KEYS[self.index]
 
+    def get_mem_rss(self):
+        """Get the memory usage (RSS) per `ps`.
+
+        If process is stopped or `ps` is unavailable, return None.
+        """
+        if not (self.running and self.process):
+            self.log.warning("Couldn't get memory usage; process isn't running.")
+            return None
+
+        try:
+            return int(subprocess.check_output(
+                "ps h -o rss {}".format(self.process.pid), shell=True).strip())
```

Comment:
I agree that dragging in python's pstools library is *probably* overkill for this however this needs testing on at least:
- OpenBSD 
- FreeBSD
- MacOSX